rmw_security_common/include/rmw_security_common/security.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
basically what we are doing here is to move the implementation from rmw_dds_common, but it also adjusts the implementation in C. probably copyright year would be better to be updated? and else where.

```suggestion
// Copyright 2025 Open Source Robotics Foundation, Inc.
```